Dockerfile

```diff
@@ -23,4 +23,14 @@ EXPOSE 3000
 
 WORKDIR /kb/deployment/lib
 
+
+# The BUILD_DATE value seem to bust the docker cache when the timestamp changes, move to
+# the end
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vcs-url="https://github.com/kbase/staging_service.git" \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.schema-version="1.1.8" \
+      us.kbase.vcs-branch=$BRANCH \
+      maintainer="Steve Chan sychan@lbl.gov"
```

Comment:
Should have really put yourself as the maintainer for it. But my name is on all of the rest anyway.